tests/swoole_http_client_coro/set_basic_auth.phpt

```diff
@@ -0,0 +1,52 @@
+--TEST--
+swoole_http_client_coro: http client
+--SKIPIF--
+<?php require __DIR__ . '/../include/skipif.inc'; ?>
+--FILE--
+<?php
+require __DIR__ . '/../include/bootstrap.php';
+$pm = new ProcessManager;
+$pm->parentFunc = function ($pid) use ($pm) {
+    go(function () use ($pm) {
+        echo httpGetBody("http://127.0.0.1:{$pm->getFreePort()}/");
```

Comment:
use http://httpbin.org (HTTPBIN_SERVER_HOST, HTTPBIN_SERVER_PORT), process manager is unnecessary